circuit/program/src/data/plaintext/to_bits.rs

```diff
@@ -35,15 +37,17 @@ impl<A: Aleo> ToBits for Plaintext<A> {
             Self::Struct(members, bits_le) => {
                 // Compute the bits of the struct.
                 let bits = bits_le.get_or_init(|| {
-                    let mut bits_le = vec![Boolean::constant(false), Boolean::constant(true)]; // Variant bit.
+                    let mut bits_le = Vec::new();
+                    bits_le.extend([Boolean::constant(false), Boolean::constant(true)]); // Variant bit.
                     U8::constant(console::U8::new(members.len() as u8)).write_bits_le(&mut bits_le);
                     for (identifier, value) in members {
                         let value_bits = value.to_bits_le();
                         identifier.size_in_bits().write_bits_le(&mut bits_le);
                         identifier.write_bits_le(&mut bits_le);
                         U16::constant(console::U16::new(value_bits.len() as u16)).write_bits_le(&mut bits_le);
-                        bits_le.extend_from_slice(&value_bits);
+                        bits_le.extend(value_bits);
```

Comment:
note: `Vec::extend` is preferable here, as it avoids cloning the members of of `value_bits`.